packages/manager/modules/telecom-universe-components/src/telecom/telephony/abbreviatedNumbers/import/telecom-telephony-abbreviated-numbers-import.modal.html

```diff
@@ -81,19 +81,16 @@
 
     <div class="modal-body"
          data-ng-if="AbbreviatedNumberModal.loading.getFile && !AbbreviatedNumberModal.errorLoading">
-        <uib-progressbar class="progress-striped"
```

Comment:
Is the `.progress-striped` still used ? Maybe it could be removed if not

Reply:
This class is coming from `bootstrap`.

Reply:
Yes @varun257 

It’s the question from @marie-j, I think we can safely remove this class :)

Reply:
Oh ... My bad then 😅